tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -9,7 +9,7 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 *Estimated completion time: 20 minutes*
 
-This tutorial guides you through an easy way to get started with your TiDB Cloud.
+This tutorial guides you through an easy way to get started with your TiDB Cloud. You can also navigate to the [**Getting Started Tutorials**](https://tidbcloud.com/console/getting-started) page in the TiDB Cloud console to follow the tutorials step by step.
```

Comment:
If users have not signed up for TiDB Cloud yet, they will be redirected to the signup page. What about providing the Getting Started Tutorials after step 1, sinch all operations in the Getting Started Tutorials needs users to have a cluster first.